### PR TITLE
Remove Application::bootstrap() as parent class has identical code.

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: src/Application.php [hunk 1]
```diff
@@ -28,19 +28,6 @@
...
-     * Load all the application configuration and bootstrap logic.
-     *
-     * You can include any other files your application needs to bootstrap
-     * in a web context here.
-     *
-     * @return void
-     */
-    public function bootstrap()
-    {
-        require_once $this->configDir . '/bootstrap.php';
```
Comment (markstory) [inline src/Application.php]: This was on purpose. My hope was that it made how the framework was working more obvious. Perhaps we could remove the base class method?

Comment (ADmad) [inline src/Application.php]: I don't think changing how bootstraping is done would be commonly needed to warrant having this method in Application class. We could document in manual how bootstraping is done with code example showing how to override this method.

Comment (markstory) [inline src/Application.php]: ok.

**Comment by ADmad** [comment]
I also added test for `Application` as suggested in #395.

**Pull request merged**